src/utils/schemaPrinter.ts

```diff
@@ -89,15 +103,61 @@ type Options = {
   sortInterfaces?: boolean;
   sortUnions?: boolean;
   sortEnums?: boolean;
+  sortCustom?: SortTC;
+  sortCustomOptions?: SortTCOptions;
```

Comment:
I don't feel that the addition of two almost the same options is a good idea. Moreover, it's unclear for users how will work `toSDL({ sortCustom: ..., sortCustomOptions: ...})` if provided simultaneously these args.

I think that it will be better to create `sortTypes` which combine custom function and predefined sorters:
```ts
sortTypes?: 
  | boolean // true by default, false disable sorting
  | 'ALPHABETIC' // same as true
  | 'ALPHABETIC_1' // Root, Scalar, Enum, Union, Interface, Object, Input (better order from my opinion)
  | 'ALPHABETIC_2' // Scalar, Root, Enum, Interface, Input, Object, Union (better order from your opinion)
  | (tc1: NamedTypeComposer<any>, tc2: NamedTypeComposer<any>) => -1 | 0 | 1; // any custom order which wants end-user     
```

If developers want to use a build-in custom sort function then they will use the following code:
```ts
schemaComposer.toSDL({
  sortTypes: 'ALPHABETIC_2', // will sort as you suggest in PR
})
```

Reply:
In such case your custom order function may be implemented in the following manner:

```ts
function createAlphabetic2(sc: SchemaComposer, optionsIfNeeded: ...) {
  const rootTypes = [];
  if (sc.has('Query')) rootTypes.push(sc.getOTC('Query').getTypeName());
  if (sc.has('Mutation')) rootTypes.push(sc.getOTC('Mutation').getTypeName());
  if (sc.has('Subscription')) rootTypes.push(sc.getOTC('Subscription').getTypeName());
    
  return (tc1, tc2) => {
    const a = sortGetPositionOfType(tc1);
    const b = sortGetPositionOfType(tc2);
    
    if (a < b) return -1;
    else if (a > b) return 1;
    else return tc1.getTypeName().localeCompare(tc2.getTypeName());
  };
}
```